moveit_core/planning_scene/src/planning_scene.cpp

```diff
@@ -34,18 +34,22 @@
 
 /* Author: Ioan Sucan */
 
-#include <boost/algorithm/string.hpp>
-#include <moveit/planning_scene/planning_scene.h>
-#include <moveit/collision_detection_fcl/collision_detector_allocator_fcl.h>
-#include <geometric_shapes/shape_operations.h>
 #include <moveit/collision_detection/collision_tools.h>
-#include <moveit/trajectory_processing/trajectory_tools.h>
-#include <moveit/robot_state/conversions.h>
+#include <moveit/collision_detection_fcl/collision_detector_allocator_fcl.h>
 #include <moveit/exceptions/exceptions.h>
+#include <moveit/planning_scene/planning_scene.h>
 #include <moveit/robot_state/attached_body.h>
+#include <moveit/robot_state/conversions.h>
+#include <moveit/trajectory_processing/trajectory_tools.h>
 #include <moveit/utils/message_checks.h>
```

Comment:
The downside of alphabetical ordering is of course that our own hierarchies in MoveIt are not represented in the includes. I'm ok with that, but it also doesn't make me happy :eyes: